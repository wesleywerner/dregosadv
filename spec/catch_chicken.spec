gonear valley portal
purloin trap,seed
go e
drop trap
put seed in trap
set trap
wait
wait
wait
wait
wait
pull string
take trap
look in trap
