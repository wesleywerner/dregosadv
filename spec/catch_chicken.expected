  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
  The a canoe rocks gently on the water as you travel up the river. Your
  guide sits at the other end of the craft rowing with an oar. You wonder how
  much longer this will take, you have been in this a canoe for a while now.
  You are anxious to start searching for the treasure.
  
  In the canoe you can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  You have no reason to ask for that.
  
> >  Garden
  You enter into a small garden -- whoever kept it hasn't been here for a
  very long time. The plants are growing wild and are all over the place.
  A hut is due northeast.
> >  The vegetable plants in the garden have grown wild without anybody around
  to harvest them. You notice some kind of tuber growing on the ground.
  
> >  You reach down pulling on the leafy parts of the plant, and a yam emerges
  from of the ground. You discard the leaves and keep the yam.
  
> >  Tribe
  The small tribal settlement of a dozen natives are going about their daily
  chores. Women and men are weaving plant fibers into fishing nets and ropes,
  some are tending to a firepit while others prepare pumpkins, tubers and
  fish.
  They don't seem to mind your presence as continue with their chores.
  A foot path is due west.
  A man sits nearby, carving a piece of wood.
  
> >  The Maro River
  You are in a canoe on the Maro river. Your guide is with you, rowing the
  craft.
  Your guide sits in the canoe gazing out over the moving river, the salty
  water gently lapping the sides of his craft.
> >  "You know, I will lend you my chicken trap, if you find me a yam. I think
  yam and fish will make a good meal."
  
> >  "This will do fine!" he says with a laugh. He hands you a pouch filled with
  seeds, along with the trap.
  
  "Remember to set the trap, then pull the string when the chicken is inside.
  Good luck!"
> >  Valley
  Pushing through the jungle you arrive in a little valley, rich with
  flowering orchids all around you.
  A foot path is due east.
  Some stone stairs leads upward.
  A clearing in the jungle is due south.
> >  Tribe
  The small tribal settlement of a dozen natives are going about their daily
  chores.
  A foot path is due west.
  A man sits nearby, carving a piece of wood.
  
> >  This looks like a good place to set the trap. Dropped.
  
> >  You open the trap.
  You scatter some seeds inside the trap.
  The trap door swings shut by itself.
  
> >  You prop the trap door open with the stick, and take hold of the free end
  of the string. You take a few paces back from the trap.
  
> >  Time passes.
  A hen appears from out the jungle.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  The hen enters the trap posthaste and begins her pecking frenzy.
  
> >  You give the string a quick yank, the propped stick flies out from under
  the trap door.
  The trap door swings shut by itself.
  
> >  Taken.
  
> >  In the trap is a hen.
  
> >